### PR TITLE
[castai-agent] use release namespace

### PR DESCRIPTION
**Pull request by matas-cast**

### CODE REVIEW

**Comment by KarolisL** [comment]
Wouldn't removing the field altogether, use the release namespace?

**Comment by matas-cast** [comment]
you might be right. I see that behavior in other public charts

File: charts/castai-agent/templates/rbac.yaml [hunk 1]
```diff
@@ -98,15 +98,15 @@ roleRef:
 subjects:
   - kind: ServiceAccount
     name: {{ include "castai-agent.serviceAccountName" . }}
-    namespace: {{ .Values.namespace }}
+    namespace: {{ .Release.Namespace }}
```
Comment (KarolisL) [inline charts/castai-agent/templates/rbac.yaml:101]: These are needed (related to my other comment), `metadata.namespace` are not.

**Comment by matas-cast** [comment]
Tested, unfortunately when materializing chart through code it doesn't work, will have to include

**Review by KarolisL — APPROVED** [review]

**Pull request merged**